src/openrct2/paint/sprite/Paint.Sprite.cpp

```diff
@@ -65,21 +65,23 @@ void sprite_paint_setup(paint_session* session, const CoordsXY& pos)
             }
         }
 
+        const auto entityPos = spr->GetLocation();
```

Comment:
Just checking: not `const auto&`?

Reply:
GetLocation currently returns a copy as we are not storing that just yet as CoordsXYZ